<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeeEditor</class>
 <widget class="QDialog" name="EmployeeEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>228</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>449</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Сотрудник</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QLineEdit" name="_lineName"/>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="_lineFunction"/>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="_lineSalary"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="_lSurname">
     <property name="text">
      <string>Фамилия</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="_lName">
     <property name="text">
      <string>Имя</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="_lMiddleName">
     <property name="text">
      <string>Отчество</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="_lSalary">
     <property name="text">
      <string>ЗП</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="_lineMiddleName"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="_lFunction">
     <property name="text">
      <string>Должность</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="_lineSurname">
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EmployeeEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EmployeeEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>checkLines()</slot>
 </slots>
</ui>
